analysis/study_definition.py

```diff
@@ -376,6 +582,19 @@
         on_or_before = "index_date",
         find_last_match_in_period = True,
     ),
+    ### Organ or kidney transplant
+    organ_kidney_transplant = patients.categorised_as(
+        {
+            "No transplant": "DEFAULT",
+            "Kidney": """
+                kidney_transplant
+            """,
+            "Organ": """
+                organ_transplant
```

Comment:
I think these need to be exclusive. What happens if someone has both a kidney and organ transplant?

Reply:
Good one! Will investigate if this ever happens.

Reply:
Found out that I need to use this codelist: https://www.opencodelists.org/codelist/opensafely/other-organ-transplant/79caeeee/ (it excludes kidney transplants). I'll change it accordingly.